resources/kprove.mak

```diff
@@ -82,6 +85,7 @@ $(K_REPO_DIR):
 $(KEVM_REPO_DIR):
 	git clone $(KEVM_REPO_URL) $(KEVM_REPO_DIR)
 	cd $(KEVM_REPO_DIR) \
+	    && git clean -fx \
```

Comment:
@denis-bogdanas does this have an effect? I mean, this recipe is executed only when $(KEVM_REPO_DIR) does NOT exist.

Reply:
Is this for your local copy, or for jenkins? I thought jenkins starts with a fresh workspace.

Reply:
It doesn't. We sometimes get deserialization exceptions. Especially now that we use different versions of KEVM on different branches.